coordinator-service/src/ceremony.ts

```diff
@@ -22,6 +22,19 @@ export interface ChunkData {
     contributions: ChunkContribution[]
 }
 
+export interface ChunkInfo {
```

Comment:
These interfaces aren't part of the ceremony state, but they're part of the REST API.  `coordinator.ts` is probably a better file to declare them.